shared/common-adapters/text.desktop.js

```diff
@@ -317,6 +318,11 @@ export const styles = {
     ...globalStyles.fontSemibold,
     ...sizeGroups['14']
   },
+  textBodySmallBold: {
```

Comment:
<img width="663" alt="screen shot 2016-05-25 at 11 18 07 am" src="https://cloud.githubusercontent.com/assets/594035/15551364/71c002e8-226a-11e6-882d-53456b3953d6.png">

in the files view.

I guess we could use SmallSemibold instead @cecileboucheron ?